packages/core/core/src/utils/nuxt/_proxyUtils.ts

```diff
@@ -1,13 +1,35 @@
 import { Context as NuxtContext } from '@nuxt/types';
 import merge from 'lodash-es/merge';
 import { ApiClientMethod } from './../../types';
+import { AxiosInstance } from 'axios';
 
 interface CreateProxiedApiParams {
   givenApi: Record<string, ApiClientMethod>;
   client: any;
   tag: string;
 }
 
+const getUrl = (nxtContent: NuxtContext, endpoint: string) => {
```

Comment:
Why it's content? This is not content. 😄 

```suggestion
const getUrl = (context: NuxtContext, endpoint: string) => {
```